projects/core/src/lib/components/home-page/home-page.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="home">
+	<section *ngFor="let section of sections" [class]="section.name">
+		<h2 [class]="section.name+'__title'">{{section.title}}</h2>
+		<ul [class]="section.name+'__list'" *ngIf="section.benefits">
+			<li class="benefits__benefit" *ngFor="let benefit of section.benefits">{{benefit}}</li>
+		</ul>
+		<p [class]="section.name+'__content'" *ngIf="section.content">{{section.content}}</p>
+		<div class="how-it-works__instructions" *ngIf="section.instructions">
```

Comment:
pls, rename this block
![image](https://user-images.githubusercontent.com/69210975/220901326-bee06bb7-4560-4f54-ab49-739e852732c7.png)

Reply:
done